base.py

```diff
@@ -498,10 +498,21 @@ def update_time(scene):
 class MBDynReadLog(bpy.types.Operator):
     """ Imports MBDyn nodes and elements by parsing the .log file """
     bl_idname = "animate.read_mbdyn_log_file"
-    bl_label = "MBDyn .log file parsing"
```

Comment:
Do keep the feature ideas coming. My proposal is basically writing itself . :)

I don't quite understand your third paragraph. We do the first action without input from the user?